src/IJulia/handle_msg.jl

```diff
@@ -21,6 +21,9 @@ function handle_msg{T}(w::Button{T}, msg)
     end
 end
 
+# keys2idx(x::OptionDict, ks) = findin(keys(s.options), ks)
+idxs2keys(x::Interact.OptionDict, indexes) = collect(keys(s.options))[indexes .+ 1]
```

Comment:
I think that the s.options is causing a bug here.

Reply:
Changing `s.options` to `x` makes it all better. I have already tested it.

Reply:
Nice one. I got (un)lucky, I was using your code to test it:
```
s = selection([:x, :y], multi=true, value=:y)
display(s)
signal(s)
```
so it was picking up the `s` from there.